fe1-web/src/features/home/README.md

```diff
@@ -0,0 +1,9 @@
+# Home
+
+Provides UI while the user is not connected to a LAO. Currently this includes a screen for showing previously connected LAOs, the component for which is provided by the `lao` feature. Moreover there are is a screen for launching new LAOs, one enabling scanning QR codes for connecting to LAOs and one for manually connecting to a LAO.
```

Comment:
This line might be a bit too wide compared to other `README`s.

Reply:
Opps, you are totally right. Forgot to run the re-wrapping function of the IDE 😅